src/main/java/com/google/sps/data/Initiative.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+public final class Initiative {
+
+    private final String title;
+    private final String location;
+    private final String startDate;
+    private final String endDate;
```

Comment:
Could you store these two attributes as Date objects?

Reply:
Ok so the reason for this is that I am storing everything in a big list of list<string>, if I change those 2 to Date type, I would need to refactor my whole code to be able to store every field that comes on each submission of the form. This is why I left it this way, but I was thinking (in some time after we get the mvp done) I could do some date parser to change those to Date types to then display them in a better format than just MM/DD/YYYY. I will keep a note of this!